src/libs/ajax/data-table-providers/WdsDataTableProvider.test.ts

```diff
@@ -457,6 +457,90 @@ describe('WdsDataTableProvider', () => {
 
       expect(actual).toStrictEqual(expected);
     });
```

Comment:
the following test case does NOT directly test any of the runtime changes in this PR. However, our initial hypothesis about the root cause of the bugs was a problem in translation between WDS formats and Entity Service formats. This unit test adds coverage for that specific area. I started by writing this unit test, and when it passed, I started to look elsewhere for a fix. But I thought I'd leave this test in place, since it adds coverage.